plugins/woocommerce/tests/e2e-pw/tests/merchant/products/block-editor/create-variable-product-block-editor.spec.js

```diff
@@ -0,0 +1,188 @@
+/* eslint-disable playwright/no-conditional-in-test */
+const { test } = require( './block-editor-fixtures' );
+const { expect } = require( '@playwright/test' );
+
+const { clickOnTab } = require( '../../../../utils/simple-products' );
+
+const NEW_EDITOR_ADD_PRODUCT_URL =
+	'wp-admin/admin.php?page=wc-admin&path=%2Fadd-product&tab=variations';
+
+const isTrackingSupposedToBeEnabled = !! process.env.ENABLE_TRACKING;
+
+const productData = {
+	name: `Variable product Name ${ new Date().getTime().toString() }`,
+	summary: 'This is a product summary',
+};
+
+const attributesData = {
+	name: 'Size',
+	options: [ 'Small', 'Medium', 'Large' ],
+};
+
+test.describe( 'Variations tab', () => {
+	test.describe( 'Variation options form', () => {
+		test( 'renders each block without error', async ( { page } ) => {
+			await page.goto( NEW_EDITOR_ADD_PRODUCT_URL );
+			await clickOnTab( 'Variations', page );
+			await page
+				.getByRole( 'heading', { name: 'Variation options' } )
+				.isVisible();
+
+			await expect( page.locator( '.block-editor-warning' ) ).toHaveCount(
+				0
+			);
+		} );
+	} );
+
+	test.describe( 'Create variable product', () => {
+		test.skip(
+			isTrackingSupposedToBeEnabled,
+			'The block product editor is not being tested'
+		);
+
+		test( 'can create a variation option and publish the product', async ( {
+			page,
+		} ) => {
+			await page.goto( NEW_EDITOR_ADD_PRODUCT_URL );
+			await clickOnTab( 'General', page );
+			await page
+				.getByPlaceholder( 'e.g. 12 oz Coffee Mug' )
+				.fill( productData.name );
+			await page
+				.locator(
+					'[data-template-block-id="basic-details"] .components-summary-control'
+				)
+				.last()
+				.fill( productData.summary );
+
+			try {
+				await page
+					.getByLabel( 'Close Tour' )
+					.click( { timeout: 5000 } );
+			} catch ( e ) {
+				console.log( 'Tour was not visible, skipping.' );
```

Comment:
The tour won't be visible after closing it. That will be the case when we run the E2E tests locally more than once.

Reply:
You should not need to close the tour, it's already disabled in the `page` fixture from `./block-editor-fixtures`.

Reply:
Good call @adimoldovan. I just found [the PR](https://github.com/woocommerce/woocommerce/pull/44699) that does that. It was merged when I started to work on mine 😄
I will add to the `block-editor-fixtures` the code to disable the variation's tour too.